### DISCUSSION TITLE
USB capture speed?

### DISCUSSION THREAD

**Discussion by doblerguy** [hyperion-project/hyperion.ng#1796] [Q&A]
Trying to understand capability/limitation of the usb capture function. I’ve got a 4k usb capture card that works just fine but I’m curious why I  can’t select 60fps? The “fastest” fps I can select, regardless of resolution, is only 30fps and that’s limited to 720x480. Shouldn’t I be able to select 1080p and 60fps?

thanks in advance!

**Comment by Lord-Grey** [comment]
The screen in Hyperion only shows what the grabber presents to the OS, i.e. the grabber itself only supports 30fps.
You might want to check the grabber‘s specification and check what you got is what you bought.

**Comment by doblerguy** [comment]
Thanks. I figured it out. It's a format issue.  I can get 60fps on MJPEG but only 30fps for YUYV
